modules/service-fips.sh

```diff
@@ -50,6 +54,69 @@ fips_disable() {
     not_supported 'Disabling FIPS'
 }
 
+fips_updates_enable() {
+    local token="$1"
+    local bypass_prompt="$2"
+    local fips_configured=0
+    local fips_kernel_version=0
+    local result=0
+
+    check_token "$FIPS_UPDATES_REPO_URL" "$token"
```

Comment:
I think the token check should come after the `_fips_updates_is_enabled` check. It's a network check, it takes a while, and has no benefit in the case of fips updates being enabled already:
```
ubuntu@xenial-ua:~$ sudo ua enable-fips-updates <redacted>
Checking token... OK
FIPS-UPDATES repository is already enabled.
```

Reply:
The `fips_enable()` function in the same file also does the token check after the is-already-enabled one.

Reply:
fixed